src/SourceGenerators/XamlGenerationTests/CornerRadiusParse.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<UserControl x:Class="XamlGenerationTests.Shared.CornerRadiusParse"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 mc:Ignorable="d">
+
+	<Grid>
+		<Border CornerRadius="12" />
+		<Border CornerRadius="12,14" />
+		<Border CornerRadius="12 14" />
```

Comment:
Is this working on Windows?  because that's not documented...

https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.cornerradius

Reply:
You are right, it does parse single value properly, but fails with two. It threw error in runtime only. I removed the "two-value" tests.